build.gradle

```diff
@@ -1,7 +1,7 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 plugins {
-    id 'com.android.application' version '8.0.2' apply false
+    id 'com.android.application' version '8.1.0' apply false
```

Comment:
Let's wait for 8.1.1 to be out so we can avoid breaking Reproducible Builds.

Reply:
8.1.1 seems to be out!

No changelogs yet though, so let's hope it includes the RB fix. If not, we can include a workaround :)

Reply:
Lacking useful release notes, I decompiled the `sdk-common` dependency for both AGP 8.1.0 and 8.1.1 and it looks like the fix made it into both, so that's not a blocker for getting this merged :)